code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -8,6 +8,8 @@ What are the archived variables for?
 
 /datum/gas_mixture
 	/// Never ever set this variable, hooked into vv_get_var for view variables viewing.
+	var/list/gases = list()
+	var/temperature = 0 //kelvins
```

Comment:
Both of these seem to be completely unused and if they are, i can probably find you a getter proc or something else, i'm pretty sure we should have one

Reply:
Let me try removing them. I initially brought them over from hyper when I was bringing it over than I swapped over to Splurt when I realized they had a more up to date version.

Reply:
removed em, no issues other than one with the auxtools check I'm trying to figure out since that only started once I updated.